src/tests/topp/ProteomicsLFQ_3_out.csv

```diff
@@ -1,88 +1,69 @@
 ProteinName,PeptideSequence,PrecursorCharge,FragmentIon,ProductCharge,IsotopeLabelType,Condition,BioReplicate,Run,Fraction,Intensity,Reference
-P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,1,1,2,2,1.54652e08,"BSA1_F2.mzML"
-P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,2,2,3,1,8.861719e07,"BSA2_F1.mzML"
-P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,3,3,5,1,9.72858e07,"BSA3_F1.mzML"
-tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,1,1,2,2,7625.908203,"BSA1_F2.mzML"
-tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,2,2,3,1,8041.891602,"BSA2_F1.mzML"
+P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,1,1,2,2,1.546597e08,"BSA1_F2.mzML"
+P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,2,2,3,1,8.861572e07,"BSA2_F1.mzML"
+P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,3,3,5,1,9.728398e07,"BSA3_F1.mzML"
+tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,1,1,2,2,7877.708008,"BSA1_F2.mzML"
+tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,2,2,3,1,6914.940918,"BSA2_F1.mzML"
 P02769|ALBU_BOVIN,C(Carbamidomethyl)C(Carbamidomethyl)TESLVNR,2,NA,0,L,1,1,1,1,2.056782e07,"BSA1_F1.mzML"
-P02769|ALBU_BOVIN,C(Carbamidomethyl)C(Carbamidomethyl)TESLVNR,2,NA,0,L,2,2,3,1,1.103722e07,"BSA2_F1.mzML"
-P02769|ALBU_BOVIN,C(Carbamidomethyl)C(Carbamidomethyl)TESLVNR,2,NA,0,L,3,3,5,1,1.794934e06,"BSA3_F1.mzML"
-P02769|ALBU_BOVIN,CC(Carbamidomethyl)TESLVNR,2,NA,0,L,3,3,5,1,2.029336e05,"BSA3_F1.mzML"
-P02769|ALBU_BOVIN,DDPHAC(Carbamidomethyl)YSTVFDK,3,NA,0,L,2,2,4,2,1.087751e07,"BSA2_F2.mzML"
```

Comment:
Hmm this was completely lost.

Reply:
Any idea why? Should not be related to the setting PATH issue, or?

Reply:
should be resolved with Hendriks fix.